swagger.yaml

```diff
@@ -5008,6 +5057,19 @@ definitions:
         format: uuid
         description: The id of the OCP cluster, that hosts will be added to
 
+  finalizing-stage:
+    description: Cluster finalizing stage managed by controller
+    type: string
+    enum:
+    - 'wait-for-finalizing'
+    - 'wait-for-cluster-operators'
+    - 'add-router-ca'
+    - 'wait-for-olm-operators'
+    - 'apply-manifests'
+    - 'wait-for-olm-operators-csv'
+    - 'done'
```

Comment:
Any reason not to align these with host-stage? E.g., "Waiting for finalizing" ("ing" verb, spaces instead of hyphens, capital letter)